src/routes/entry-point.ts

```diff
@@ -1,13 +1,15 @@
 import Router from '@koa/router';
 import { Context, Middleware, Next } from 'koa';
 import { schema } from 'rdf-namespaces';
-import url from 'url';
 import Routes from './index';
 
 export default (router: Router): Middleware => (
   async ({ request, response }: Context, next: Next): Promise<void> => {
     response.body = {
-      '@id': url.resolve(request.origin, router.url(Routes.EntryPoint, {})),
+      '@context': {
+        '@base': request.origin,
```

Comment:
can't find where I encountered this, but seems to satisfy the same cases as the `<base>` HTML tag